gitcoins-cordapp/workflows/src/main/kotlin/com/gitcoins/flows/QueryGitUserDatabaseFlow.kt

```diff
@@ -0,0 +1,25 @@
+package com.gitcoins.flows
+
+import com.gitcoins.schema.GitUserMappingSchemaV1
+import net.corda.core.flows.FlowException
+import net.corda.core.flows.FlowLogic
+import net.corda.core.flows.StartableByRPC
+
+/**
+ * Utility flow to query the [GitUserMappingSchemaV1.GitUserMapping] table.
```

Comment:
This should be an utility function, notice that you don't contact counterparty anywhere here.

Reply:
Let's discuss this one in person.